db/fixtures/miq_policies.yml

```diff
@@ -0,0 +1,29 @@
+---
+- :name: analyze_new_images
+  :description: Analyze all new images
```

Comment:
This description and name might want to say "container" somewhere in it, so it's clear it only pertains to container images and not, say, cloud images.